packages/router/src/create_url_tree.ts

```diff
@@ -75,7 +79,7 @@ class Navigation {
       throw new Error('Root segment cannot have matrix parameters');
     }
 
-    const cmdWithOutlet = commands.find(c => typeof c === 'object' && c != null && c.outlets);
+    const cmdWithOutlet = commands.find(c => isCommandWithOutlets(c));
```

Comment:
What do you think of this style? (as the arrow function is un-needed here.)
```suggestion
    const cmdWithOutlet = commands.find(isCommandWithOutlets);
```